controllers/shooter.py

```diff
@@ -60,9 +67,6 @@ def tracking(self) -> None:
         ):
             self.next_state("firing")
 
-        # Reset each loop so that the call has to be made each control loop
```

Comment:
I've removed this comment wholesale as I think it should be obvious from the `will_reset_to` usage, but if anyone else thinks otherwise, I can move it.